soccer/src/soccer/strategy/agent/position/seeker.cpp

```diff
@@ -207,9 +190,15 @@ double Seeker::eval_point(rj_geometry::Point ball_pos, rj_geometry::Point curren
         block_shot_loss = 1;
     }
 
+    double min_seeker_dist = std::numeric_limits<double>::infinity();
```

Comment:
can we add a comment here explaining what is happening (with context to the communication)

Reply:
Also make the other heuristics in this method use infinity as a starting point instead of some large number